src/cellranger/processes/count.nf

```diff
@@ -27,6 +27,24 @@ def generateCellRangerCountCommandDefaults = {
 	} else if (processParams.containsKey('chemistry')) {
 		_chemistry = processParams.chemistry
 	}
+
+	_includeIntrons = null
+	if(processParams.containsKey('includeIntrons')) {
+		if (processParams.includeIntrons == true) {
+			_includeIntrons = true
+		} else {
+			_includeIntrons = false
+		}
+	}
```

Comment:
This can be simplified by:
```
_includeIntrons = processParams?.includeIntrons
```
And actually, you could even consider to use any of this local variables

Reply:
As above